spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  describe 'content of user index page' do
+    let(:user1) { User.create(name: 'Juwairiyya', photo_link: 'photo_url', bio: 'text', posts_counter: 0) }
+
+    before(:each) do
+      user1.posts.create(title: 'Post Title', text: 'Post Content')
+      visit "/users/#{user1.id}"
+    end
+
+    it 'shows the username' do
+      expect(page).to have_content(user1.name)
+    end
+
+    it 'should show users photo' do
+      expect(page).to have_css("img[src='#{user1.photo_link}']")
+    end
+
+    it 'should display user name' do
+      expect(page).to have_content(user1.name)
+    end
```

Comment:
- [ ] This is duplicated as i can see it above also. Kindly make sure only one is available 👏